_includes/manuals/1.15/4.5.2_cli_success.md

```diff
@@ -56,9 +56,6 @@ hammer template update --id 4 --file /tmp/ks.txt
 <pre>
 hammer template update --id 1 --operatingsystem-ids 1
 </pre>
-
```

Comment:
It's not missing anymore, however I don't think it's really necessary to showcase the command here.